.github/workflows/stable.yml

```diff
@@ -26,3 +27,8 @@ jobs:
 
       - name: Run tests
         run: bundle exec rake
+        if: matrix.ruby != 'truffleruby'
+
+      - name: Run specs (truffleruby)
+        run: bundle exec rake spec
```

Comment:
The reason is `bundle exec rake` runs `rubocop spec cucumber`.
`rubocop` is pointless to run again here.
`cucumber` fails because it tries to use `fork` (most likely `childprocess` tries to).
IIRC that's fixed in latest cucumber/aruba, but updating them seems out of scope of this PR.

Reply:
Agree on rubocop but that's for another day to fix/make this faster

Reply:
Having those comments in the code may be nice for our future selves :sweat_smile: I might just add them later though.

Reply:
Yeah, fair enough, I'll add a comment here

Reply:
Done, should be all good and ready to merge now, thanks for the review!

Reply:
Thanks for all your working and fixing this up, sorry I wasn't around